src/styles/blocks/antisnake.scss

```diff
@@ -0,0 +1,31 @@
+.antisnake {
+  display: grid;
+  justify-content: center;
```

Comment:
hm, as I understand 'justify-content' is useless when we stretch our blocks.

Reply:
So let's remove this rule ?